vi/lessons/basics/pipe-operator.md

```diff
@@ -1,70 +1,69 @@
 ---
 layout: page
-title: Pipe Operator 
+title: Pipe Operator
```

Comment:
@doomspork

Yeah, this is our purpose. Our team discussed while doing the translation, and we decided to not translate some words.